.github/workflows/lint.yml

```diff
@@ -86,6 +86,21 @@ jobs:
         run: |
           echo "::remove-matcher owner=mypy::"
 
+      - name: lint pyproject.toml
+        if: ${{ !cancelled() && (steps.deps.outcome == 'success') }}
+        run: |
+          make toml-lint
```

Comment:
Why isn't this done through tox like all the other lint operations?

Reply:
No particular reason, let me add it.